action.yml

```diff
@@ -12,6 +12,10 @@ inputs:
   format:
     description: "The output format (commonmark or html)"
     default: "commonmark"
+  base-url:
+    description: "The base URL for all relative URLs generated in documents"
+    required: false
+    default: "./"
```

Comment:
Regardless of what happens with `Page.swift`, I'm definitely happy to have `base-url` exposed in the action interface.